database/sampleDataMaker.js

```diff
@@ -2,7 +2,7 @@
 const fs = require('fs');
 
 const {
-  type, pageNum, publisher, dates, title, isbn, language, characterArr, awardsArr, editionsArr, settingsArr,
+  type, pageNum, publisher, dates, title, isbn, language, characterArr, awardsArr, editionsArr, settings,
```

Comment:
Interesting use case - the way you're importing these functions. I would have done it differently.